frontend/src/components/TracksTile.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { FC } from 'react';
+import { ReactComponent as ThreeDots } from '../assets/icons/more-horizontal.svg';
+import { ReactComponent as UsersIcon } from '../assets/icons/users.svg';
+
+// set properties for tracktile
```

Comment:
try and avoid comments which state the obvious. In general a comment should only be needed where there is complex code that is hard to understand